release notes/v0.36.0.md

```diff
@@ -0,0 +1,76 @@
+k6 v0.36.0 is here! 🎉 It introduces a couple new features which enhance its usability, and includes a bunch of fixes and the result of refactoring ongoing efforts.
+
+In total, we have closed 27 issues.
+
+## New Features!
+
+### Sourcemaps support ([#2082](https://github.com/grafana/k6/pull/2082))
+
+// TODO: Ask @MStoykov for support on this
+
+### Ability to abort tests ([#2093](https://github.com/grafana/k6/pull/2093))
+
+Thanks to the contribution of [@gernest](https://github.com/gernest), K6 now has the ability to abort a test. [#2093](https://github.com/grafana/k6/pull/2093) made the `k6/execution` exposes the `test.abort()` function, which allows to immediately abort a K6 load test run with the exit code `108`.
+
+Aborting is possible during initialization:
+```javascript
+import exec from "k6/execution";
+exec.test.abort();
+```
+
+As well as inside the default function:
+```javascript
+import exec from "k6/execution";
+
+export default function() {
+  // Note that you can abort with a specific message too
+  exec.test.abort("this is the reason");
+}
+
+export function teardown() {
+  console.log("teardown will still be called after test.abort()");
+}
+```
+
+Or during the VU initialization:
+```javascript
+import exec from "k6/execution";
+
+// This won't fail on initial parsing of the script, but on VU initialization.
+if (__VU == 1) {
+  exec.test.abort();
+}
+
+export default function() {}
+```
+
+### K6 inspect extended output ([#2279](https://github.com/grafana/k6/pull/2279))
+
+Following [#2279](https://github.com/grafana/k6/pull/2279), the `k6 inspect` command now supports an `--execution-requirements` flag. When used, the command's output will include fields related to the execution requirements, the configuration K6 derived from the execution context, and will also expose the `maxVUs` and `totalDuration` fields.
+
+
+### Forcing HTTP/1 protocol ([#2222](https://github.com/grafana/k6/pull/2222))
+
+Thanks to the work of [@sjordhani22](https://github.com/sjordhani22), [#2222](https://github.com/grafana/k6/pull/2222) made it possible to force K6 to use version 1.1 of the protocol when firing HTTP requests. 
```

Comment:
```suggestion
Thanks to the work of @sjordhani22, [#2222](https://github.com/grafana/k6/pull/2222) made it possible to force K6 to use version 1.1 of the protocol when firing HTTP requests. 
```
GitHub should resolve the mention automatically

Reply:
I prefer to leave it as a plain Markdown link if that's okay with you. It both ensures that the link is present in the repo's markdown file itself. Also as a courtesy, as this might contribute to the search engine's ranking score of the person.